R/rows-dm.R

```diff
@@ -234,9 +235,18 @@ check_keys_compatible <- function(x, y) {
   # FIXME
 }
 
+get_dm_rows_op <- function(operation_name) {
+  switch(operation_name,
+    "insert"   = c(fun = rows_insert, pb_label = "inserting rows"),
+    "update"   = c(fun = rows_update, pb_label = "updating rows"),
+    "patch"    = c(fun = rows_patch, pb_label = "patching rows"),
+    "upsert"   = c(fun = rows_upsert, pb_label = "upserting rows"),
+    "delete"   = c(fun = rows_delete, pb_label = "deleting rows"),
+    "truncate" = c(fun = rows_insert, pb_label = "truncating rows")
```

Comment:
A little clearer, because a list is returned:

```suggestion
    "insert"   = list(fun = rows_insert, pb_label = "inserting rows"),
    "update"   = list(fun = rows_update, pb_label = "updating rows"),
    "patch"    = list(fun = rows_patch, pb_label = "patching rows"),
    "upsert"   = list(fun = rows_upsert, pb_label = "upserting rows"),
    "delete"   = list(fun = rows_delete, pb_label = "deleting rows"),
    "truncate" = list(fun = rows_insert, pb_label = "truncating rows")
```